metagraph-plugin/katana-metagraph.yml

```diff
@@ -0,0 +1,16 @@
+# Install with:
+#   conda env create -f katana-metagraph.yml
+name: katana-metagraph
+channels:
+  - katanagraph/label/dev
+  - conda-forge
+  - metagraph
+  - insomniapx
```

Comment:
What is `insomniapx` for? I'd like to avoid pulling in extra channels we don't need, and if we do, I would like to document it if it's not obvious.

Reply:
Hi Arthur, could you assist me with uploading the conda pkg to the katanagraph's conda channel?

Reply:
I think I need a username/psw to do, right? That's how it works with my personal conda channel

Reply:
I can add you to the katanagraph org on anaconda.org and then you can upload it the same way with one additional arg which sets the target user.

Reply:
I added you. You should be able to upload package to the `katanagraph` user now. Don't upload any testing packages use your own user for that. But go ahead and upload an initial version of the metagraph-katana package.

Reply:
To confirm: Can I directly use `anaconda upload --user katanaGraph xxx.tar.bz2`? Do I need to specify the channel `katanagraph/label/dev`?

Reply:
You should specify the `dev` label. I htink it's `--label` but please check the docs.

Reply:
Yes. Thanks! I figured it out by trying multiple times. (I found anaconda users can freely delete existing packages, which is different from pypi, I think). It is `anaconda upload --user katanaGraph -c dev xxx.tar.bz2`.